alert-platform/build.gradle

```diff
@@ -0,0 +1,38 @@
+plugins {
+    id 'java-platform'
+}
+
+dependencies {
+    constraints {
+        api 'com.github.springtestdbunit:spring-test-dbunit:1.3.0'
+        api 'org.testcontainers:postgresql:1.12.5'
+        api 'org.apache.activemq:activemq-broker:5.15.10'
+
+        api 'org.jsoup:jsoup:1.9.2'
+        api 'com.jayway.jsonpath:json-path:2.3.0'
+        api 'com.sun.mail:javax.mail:1.6.2'
+        api 'javax.mail:javax.mail-api:1.6.2'
+
+        api 'org.postgresql:postgresql:42.2.8'
+        api 'org.liquibase:liquibase-core:3.8.9'
+
+        api 'com.synopsys.integration:blackduck-common:49.2.0'
+        api 'com.synopsys.integration:integration-rest:5.1.1'
+        api 'com.synopsys.integration:integration-common:23.0.0'
+
+        api 'io.springfox:springfox-swagger2:2.8.0'
+        api 'io.springfox:springfox-swagger-ui:2.8.0'
+
+        api 'org.springframework.security.extensions:spring-security-saml2-core:1.0.10.RELEASE'
+
+        api 'com.google.cloud:libraries-bom:2.2.0'
+
+        api 'com.synopsys.integration:int-jira-common:2.0.0'
+        api 'com.google.cloud:libraries-bom:2.2.0'
```

Comment:
You have this listed twice. This line and line 28

Reply:
Now line 22